xmantissa/test/test_port.py

```diff
@@ -703,11 +777,11 @@ def test_listUnlisteningFactory(self):
         "      --version           Display Twisted version and exit.\n"
         "      --help              Display this help and exit.\n"
         "\n"
-        "Delete an existing port binding from a factory. If a server is "
-        "currently running\n"
-        "using the database from which the port is deleted, the factory "
-        "will *not* stop\n"
-        "listening on that port until the server is restarted.\n"
+        "Delete an existing port binding from a factory.  If a server is "
+        "currently\n"
+        "running using the database from which the port is deleted, the "
+        "factory will\n"
+        "*not* stop listening on that port until the server is restarted.\n"
```

Comment:
Rather than reformatting it this way, maybe just collapse out all the whitespace with `" ".join(....split())`?